### PR TITLE
CRAYSAT-1587: Document sat jobstat command

### PR DESCRIPTION
**Pull request by ekamin-hpe**
* Add sat jobstat to "sat dependencies" table.
* Add sat jobstat to table describing sat authentication types.

## Summary and Scope

See commit message

## Issues and Related PRs

* Part of CRAYSAT-1587

## Pull Request Checklist

- [X] Version number(s) incremented, if applicable
- [X] Copyrights updated
- [X] License file intact
- [X] Target branch correct
- [X] CHANGELOG.md updated
- [X] Testing is appropriate and complete, if applicable
- [X] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by jack-stanek-hpe — APPROVED** [review]
Looks good to me, it seems the link checker is failing for reasons irrelevant to the contents of this PR. It might be good to patch that on integration and then merge this, IMO.

**Comment by ekamin-hpe** [comment]
> Looks good to me, it seems the link checker is failing for reasons irrelevant to the contents of this PR. It might be good to patch that on integration and then merge this, IMO.

I *think* that issue might be going away on its own very soon, actually. It's failing to reach the URL for the getting started guide: https://www.hpe.com/support/ex-S-8000

As I understand it the GSG will be getting published any day now.

**Comment by jld-dehn-hpe** [comment]
> > Looks good to me, it seems the link checker is failing for reasons irrelevant to the contents of this PR. It might be good to patch that on integration and then merge this, IMO.
> 
> I _think_ that issue might be going away on its own very soon, actually. It's failing to reach the URL for the getting started guide: https://www.hpe.com/support/ex-S-8000
> 
> As I understand it the GSG will be getting published any day now.

@ekamin-hpe, yes, I expect to update that URL likely tomorrow.

File: docs/introduction.md [hunk 1]
```diff
@@ -261,6 +261,12 @@ for each subcommand. Each service or component is listed under the product it be
 
 None
 
+### `sat jobstat`
+
+#### PBS
```
Comment (haasken-hpe) [inline docs/introduction.md:266]: Is this really only provided with PBS and not Slurm?

**Review by haasken-hpe — APPROVED** [review]

File: docs/introduction.md (hunk 1 above)
Comment (ekamin-hpe) [reply docs/introduction.md:266]: Correct, according to Vinay the state checker service is only found in the PBS product stream.

**Pull request merged**